### ISSUE TITLE
Cleanup newlines and blanks in "Fuel" packages

### ISSUE THREAD

**Issue by astares** [theseion/Fuel#258]
We cleanup newlines and blanks in methods, see https://github.com/pharo-project/pharo/issues/10396 (which is a general task)
and https://github.com/pharo-project/pharo/issues/10483 (for the specific subtask for the Fuel packages)

To fix it @theseion just needs to run the script found in https://github.com/pharo-project/pharo/issues/10483 in Pharo on latest development version and commit (also based on latest version):

```Smalltalk
badCases := SystemNavigation default allMethodsSelect: [ :each | each sourceCode trimRight size ~= each sourceCode size ].

fixInPackage := badCases select: [:each | each package name beginsWith: 'Fuel' ].

fixInPackage do: [:each |
	| refactoring classOrTrait |
	classOrTrait := each traitSource ifNil: [ each methodClass ] ifNotNil: [ each traitSource innerClass ].
	refactoring := RBAddMethodChange compile: each sourceCode trimRight in: classOrTrait for: nil.				 
	refactoring execute	
]
```

**Comment by theseion** [comment]
Thanks @astares. I've published 3.0.4 with the changes and opened a PR with Pharo.

**Comment by astares** [comment]
@theseion Thanks for caring!
